Marlin/src/pins/stm32f4/pins_LERDGE_X.h

```diff
@@ -99,16 +95,22 @@
 #define TEMP_1_PIN                          -1    // Analog Input
 #define TEMP_BED_PIN                        PC1   // Analog Input
 
+// Lergde-S can choose thermocouple/thermistor mode in software.
+// For use with thermistors, these pins must be OUT/LOW. 
```

Comment:
Random user asks: "What do I need to do to make these pins OUT/LOW?"

Reply:
Like I wrote in the update to the comment in the pins file: It is done automatically now, according to the configured thermistor. The code is in the temperature class initialization.

One can also use M42 to toggle the pin manually for testing purposes.

Reply:
I just noticed that the question was about Lerdge-X - this board does not support thermocouples.